citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.jsp

```diff
@@ -155,7 +278,26 @@ $(function() {
 		var authId = $(this).data('authority-id');
 		$("#uriAuthorLocalId").val(authId);
 		$("#authorAuthorityUsed").html("Using stored authority entry <i>" + $(this).data('authority-name') + "</i>.");
-		$("#uriLoadingFound").popover('hide');
+		$("#uriLoadingFoundEditor").popover('hide');
+		event.preventDefault();
+	});
+	
+	$("#editorIconContainer").on('click', ".popover #createAuthority", function() {
```

Comment:
the id 'createAuthority' is already used for authors and should be changed here.